assets/app/scripts/controllers/build.js

```diff
@@ -70,6 +71,15 @@ angular.module('openshiftConsole')
               }
               $scope.build = build;
             }));
+            watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
+              if (action === "DELETED") {
+                $scope.alerts["deleted"] = {
+                  type: "warning",
+                  message: "This build configuration has been deleted."
+                };
+              }
+              $scope.buildConfig = buildConfig;
```

Comment:
@spadgett so I've checked and no it returns the original bc object. But I guess that what we need sice we are checking for the annotation. right ?